gremlin-javascript/src/main/javascript/gremlin-javascript/package.json

```diff
@@ -40,4 +40,4 @@
   "engines": {
     "node": ">=6"
```

Comment:
We should state `"node": ">= 10"` in the package.json

Reply:
oops - shouldn't have missed that - fixed now